### PR TITLE
Make the Register button on home page display conditionally.

### PR DESCRIPTION
**Pull request by dcadams**
This is required to be different on suclass, where students are
auto-registered when using WebAuth, from Lagunita where all students
must go through the registration step.

A new setting of DISABLE_REGISTER_BUTTON is introduced.

### CODE REVIEW

**Comment by dcadams** [comment]
@stvstnfrd - Not required for release.

File: lms/templates/navigation.html [hunk 1]
```diff
@@ -107,7 +107,7 @@ <h1 class="logo">
     </ol>
 
     <ol class="right nav-courseware">
-      % if not settings.FEATURES['DISABLE_LOGIN_BUTTON']:
+      % if not settings.DISABLE_REGISTER_BUTTON:
```
Comment (stvstnfrd) [inline lms/templates/navigation.html]: Can you explain this change? Was the existing flag insufficient, etc.?

Comment (dcadams) [inline lms/templates/navigation.html]: You could think of it as a bug. The DISABLE_LOGIN_BUTTON setting was controlling visibility of the Register and Login buttons. We wanted finer control; remove only the Register button.

Comment (stvstnfrd) [inline lms/templates/navigation.html]: Ah, gotcha. Should this have a companion PR in secrets?

Comment (dcadams) [inline lms/templates/navigation.html]: That would be PR #127

**Comment by stvstnfrd** [comment]
+1

**Pull request merged**